kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/BigQueryWriter.java

```diff
@@ -154,6 +156,10 @@ public void writeRows(PartitionedTableId table,
           // rate limit exceeded error
           logger.warn("Rate limit exceeded for table {}, attempting retry", table);
           retryCount++;
+        } else if (err.getCode() == UNKNOWN_CODE && err.getCause() != null && err.getCause() instanceof IOException){
```

Comment:
Nit: no need for the null guard if all we're doing is an `instanceof` check (it'll handle `null` fine):
```suggestion
        } else if (err.getCode() == UNKNOWN_CODE && err.getCause() instanceof IOException){
```